### PR TITLE
Changes to the imports

### PR DESCRIPTION
**Pull request by lizgzil**
Functions are in utils, so importing needs to change slightly

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import pandas as pd
 import pickle
 import sys
-from separate import (process_reference_section,
-                      summarise_predicted_references,
-                      test_get_reference_components)
+from utils import (process_reference_section,
+                    summarise_predicted_references,
```
Comment (SamDepardieu) [inline tests.py]: This line and the next line should be aligned with the `p` of `process`

File: utils/__init__.py [hunk 2]
```diff
@@ -1,12 +1,17 @@
-from .separate import process_reference_section
-from .predict import predict_references, predict_structure
+from .separate import (process_reference_section, 
+                        summarise_predicted_references,
```
Comment (SamDepardieu) [inline utils/__init__.py]: Same here

**Comment by SamDepardieu** [comment]
@lizgzil Can you check if everything is there?

**Comment by SamDepardieu** [comment]
[Import changes](https://trello.com/c/F4AF1106/453-import-changes)

File: utils/__init__.py [hunk 3]
```diff
@@ -1,13 +1,19 @@
-from .separate import (process_reference_section, split_sections,
+from .separate import (process_reference_section,
+                       summarise_predicted_references,
+                       test_get_reference_components,
+                       split_sections,
                        split_reference)
-from .predict import predict_references, predict_structure
+from .predict import predict_references, predict_structure, test_structure
```
Comment (lizgzil) [inline utils/__init__.py]: `test_structure` is in tests.py now, shouldnt be in predict.py anymore

**Comment by SamDepardieu** [comment]
I moved everything back in the right place 👌

**Comment by lizgzil** [comment]
This code runs on my machine and outputs what is to be expected, happy to merge

**Pull request merged**